dynawo/sources/Models/Modelica/PreassembledModels/CMakeLists.txt

```diff
@@ -158,6 +158,7 @@ set(MODELICA_PREASSEMBLED_MODELS
   IECWT4ACurrentSource2020.xml
   IECWT4BCurrentSource2015.xml
   IECWT4BCurrentSource2020.xml
+  InertialGrid.xml
```

Comment:
Could you add it also in dynawo/sources/Models/Modelica/PreassembledModels/NrtModels.txt ? thanks

Reply:
Done